src/components/limit-elements/limit-elements.html

```diff
@@ -0,0 +1,14 @@
+<div class="m-limit-elements">
+    <ul class="m-limit-elements__list">
+        <li v-if="isLimitedElementDisplay"
+            v-for="(element, index) in limitedElements">{{ element }}</li>
```

Comment:
Toujours utiliser :key avec v-for 

https://vuejs.org/v2/style-guide/#Keyed-v-for-essential

et pas de v-if avec de v-for

https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential